src/dialogs/more-info/controls/more-info-cover.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
+
+import { attributeClassNames } from "../../../common/entity/attribute_class_names";
+import { featureClassNames } from "../../../common/entity/feature_class_names";
+
+import CoverEntity from "../../../util/cover-model";
+
+import "../../../components/ha-attributes";
+import "../../../components/ha-cover-tilt-controls";
+import "../../../components/ha-labeled-slider";
+
+import type { HassEntity } from "home-assistant-js-websocket";
+import type { HomeAssistant } from "../../../types";
+
+const FEATURE_CLASS_NAMES = {
+  4: "has-set_position",
+  128: "has-set_tilt_position",
+};
+
+@customElement("more-info-cover")
+class MoreInfoCover extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) stateObj!: HassEntity;
+
+  @internalProperty() private _entityObj?: CoverEntity;
+
+  protected updated(changedProperties: PropertyValues): void {
+    super.updated(changedProperties);
+
+    if (changedProperties.has("stateObj")) {
+      this._entityObj = new CoverEntity(this.hass, this.stateObj);
+    }
+  }
+
+  protected render(): TemplateResult {
+    if (!this._entityObj) {
+      return html``;
+    }
+
+    return html`<div class=${classMap(this._computeClassNames())}>
```

Comment:
I'm not sure how to convert this computeClassNames over

Reply:
Looks like we don't need all those classes, it is either `max-height: 0px;` or `max-height: 280px;`

So make a `classMap` with 2 classes and the right checks <https://github.com/home-assistant/frontend/pull/7722/files#diff-dca629d25d790b59918244ca041907b2bc24c7b89de8f7346020e710a6191ad2R110-R121>